holidays/countries/turkmenistan.py

```diff
@@ -0,0 +1,69 @@
+from holidays.calendars.gregorian import JAN, MAR, MAY, SEP, OCT, DEC
+from holidays.groups import InternationalHolidays, IslamicHolidays
+from holidays.holiday_base import HolidayBase
+
+
+class Turkmenistan(HolidayBase, InternationalHolidays, IslamicHolidays):
+    """
+    Turkmenistan holidays.
+    References:
+        https://en.wikipedia.org/wiki/Public_holidays_in_Turkmenistan
+        https://www.timeanddate.com/holidays/turkmenistan/
+    """
+
+    country = "TM"
+    start_year = 1991 
```

Comment:
Maybe consider moving this to 1992 instead to avoid overlapping with USSR ones and simplify holidays assignment:
```suggestion
    start_year = 1992
```